logs-shipper/src/main/java/com/Resource.java

```diff
@@ -2,33 +2,54 @@
 
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
+import org.elasticsearch.action.ActionListener;
+import org.elasticsearch.action.index.IndexRequest;
+import org.elasticsearch.action.index.IndexResponse;
+import org.elasticsearch.action.search.SearchRequest;
+import org.elasticsearch.action.search.SearchResponse;
+import org.elasticsearch.client.*;
+import org.elasticsearch.index.query.BoolQueryBuilder;
+import org.elasticsearch.index.query.QueryBuilders;
+import org.elasticsearch.search.SearchHit;
+import org.elasticsearch.search.SearchHits;
+import org.elasticsearch.search.builder.SearchSourceBuilder;
 
 import javax.inject.Inject;
 import javax.inject.Singleton;
-import javax.ws.rs.GET;
-import javax.ws.rs.Path;
-import javax.ws.rs.Produces;
+import javax.ws.rs.*;
 import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.util.HashMap;
+import java.util.Map;
 import java.util.UUID;
 
 @Singleton
-@Path("bootcamp")
+@Path("api")
 public class Resource {
 
     private LogsConfiguration logsConfiguration;
     private String seed;
 
     public static int VISITS_COUNTER = 1;
+    public static int ID = 1;
+
+    private RestHighLevelClient client;
```

Comment:
Our convention is to use `final`, as it should be assigned only one time

Reply:
wait, what should be final? they are assigned once, but are being incremented in places around the code.

Reply:
deleted the ID. don't need it anymore.